mkdocs.yml

```diff
@@ -0,0 +1,171 @@
+site_name: Settle OAuth with GCP
+site_url: https://SettleAPI.github.io/settle-oauth-gcp
+site_author: Settle <support@settle.eu>
+repo_url: https://github.com/SettleAPI/settle-oauth-gcp
+repo_name: SettleAPI/settle-oauth-gcp
+edit_uri: edit/master/docs
+docs_dir: docs
+site_dir: site
+# gh-deploy command settings
+remote_branch: gh-pages
+remote_name: origin
+
+use_directory_urls: false
+
+theme:
+  name: material
+  language: en
+  palette:
+    - scheme: default
+      toggle:
+        # icon: material/toggle-switch-off-outline
+        icon: material/weather-sunny
+        name: Switch to dark mode
+    - scheme: slate
+      primary: blue
+      accent: orange
+      toggle:
+        # icon: material/toggle-switch
+        icon: material/weather-night
+        name: Switch to light mode
+  font:
+    text: Roboto
+    code: Roboto Mono
+  features:
+    - navigation.indexes
+    - navigation.top
+
+# If nothing is specified, navigation will be implicitly created using
+# capitalized direcotry names and the H1 of each file, sorted alphanumerically
+# used to be done using 'pages', now deprecated, new syntax is 'nav'
+#nav:
+#  - Home: 'install.md'
+
+markdown_extensions:
+  # https://squidfunk.github.io/mkdocs-material/extensions/pymdown/
+  # see link above for all available extensions
+
+  # https://squidfunk.github.io/mkdocs-material/extensions/admonition/
+  # add block-styled side content, e.g. summaries, notes, hints or warnings
+  - markdown.extensions.admonition:
+
+  # https://facelessuser.github.io/pymdown-extensions/extensions/details/
+  # support collapsible blocks, might not compatible with IE and Edge
+  - pymdownx.details:
+
+  # https://python-markdown.github.io/extensions/toc/
+  - markdown.extensions.toc:
+      # https://squidfunk.github.io/mkdocs-material/extensions/permalinks/
+      # inserts anchor at end of each headline to provide link to a subpart of the document
+      permalink: true
+      # keep at 1 for H1 in document to be the title when specified in mkdocs.yml
+      # if set to 2 pushes H1 to H2, if set to 3 pushes H1 to H3
+      # baselevel: 1
+
+  # https://facelessuser.github.io/pymdown-extensions/extensions/tabbed/
+  - pymdownx.tabbed:
+
+  # https://squidfunk.github.io/mkdocs-material/extensions/footnotes/
+  - markdown.extensions.footnotes:
+
+  # https://python-markdown.github.io/extensions/abbreviations/
+  - markdown.extensions.abbr:
+
+  # https://python-markdown.github.io/extensions/definition_lists/
+  - markdown.extensions.def_list:
```

Comment:
OMG :heart_eyes: I've been missing DLs for a while

Reply:
By using snippets, one can insert a master definition list into every file with just three lines, and then all your favorite acronyms would have an on hover definition!

Reply:
ZOMG :exploding_head: